config.rdf

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
+<rdf:RDF 
+     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
+     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
+
+    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
+        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
+        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
+    </owl:Ontology>
+
+    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
+        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
+        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
```

Comment:
Well, my opinion is obviously for modularity.

On Friday, January 16, 2015, Andrew Woods <notifications@github.com> wrote:

> In config.rdf
> <https://github.com/fcrepo4/ontology/pull/13#discussion-diff-23088144>:
>
> > +<?xml version="1.0"?>
> > +<?xml-stylesheet type="text/xsl" href="owl2html.xsl"?>
> > +<rdf:RDF
> > +     xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
> > +     xmlns:owl="http://www.w3.org/2002/07/owl#"
> > +     xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
> > +     xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
> > +
> > +    <owl:Ontology rdf:about="http://fedora.info/definitions/v4/config#">
> > +        <rdfs:label xml:lang="en">Fedora Commons Configuration Ontology</rdfs:label>
> > +        <rdfs:comment xml:lang="en">Ontology for user-defined configuration properties and classes on Fedora-curated resources.</rdfs:comment>
> > +    </owl:Ontology>
> > +
> > +    <owl:DatatypeProperty rdf:about="http://fedora.info/definitions/v4/config#hasIndexingTransformation">
> > +        <rdfs:label xml:lang="en">has indexing transformation</rdfs:label>
> > +        <rdfs:domain rdf:resource="http://fcrepo.info/definitions/v4/repository#Container"/>
>
> We need to get resolution on this. The question on the table is:
> Do we have one namespace that defines the predicates that users can update
> on resources, or do we have multiple such namespaces?
> Does the potential clarity provided by multiple namespaces outweigh the
> maintenance issues and the potential user-confusion that may be created in
> having to look for the right namespace?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fcrepo4/ontology/pull/13/files#r23088144>.
>